lib/ui/screens/protected/resturant-store/restaurant/index.tsx

```
@@ -0,0 +1,349 @@
+            </div>
+          </div>
+        </PaddingContainer>
+
+        <Spacer height="20px" />
+
+        {/* Main Section */}
+
+        <PaddingContainer>
+          {menuData.map((category, index) => (
+            <div className="mb-4 p-3" id={category.href}>
+              <h2 className="mb-4 font-inter text-gray-900 font-bold text-2xl sm:text-xl leading-snug tracking-tight">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)